packages/apidom-ls/src/services/validation/linter-functions.ts

```diff
@@ -1068,4 +1069,41 @@ export const standardLinterfunctions: FunctionItem[] = [
       return true;
     },
   },
+  {
+    functionName: 'apilintOpenAPIParameterFieldIsDefinedWithinPathTemplate',
```

Comment:
What would you say for the following simplified name?

```suggestion
    functionName: 'apilintOpenAPIParameterInPathTemplate',
```